### PR TITLE
Generalize csv import

### PR DESCRIPTION
**Pull request by Tilana**
Generalize csv import such that text data and optionally labels from any csv file are loadable into MongoDB.
For this specifying the csv path, the text column and label column is necessary.

### CODE REVIEW

**Comment by Tilana** [comment]
I further added a sharedId property to the *classification_sample* model.
This enables to export the classified documents and then match these, based on the sharedId, with other properties from a different csv file.

File: app/classifier.py [hunk 1]
```diff
@@ -219,9 +219,10 @@ def _classify_probs(self, seqs: List[str]) -> List[Dict[str, float]]:
             return []
         # Split very large requests into chunks since
         # (intermediate) bert data is huge.
-        if len(seqs) > 1000:
-            return (self._classify_probs(seqs[:1000]) +
-                    self._classify_probs(seqs[1000:]))
+        BATCH_SIZE = 200
```
Comment (bdittes) [inline app/classifier.py]: please put constants at the top of the file

File: local.py [hunk 2]
```diff
@@ -72,36 +86,50 @@ def outputCsv(c: classifier.Classifier) -> None:
...
-                existing.training_labels = [
-                    dict(topic=l) for l in training_labels
-                ]
-                existing.use_for_training = len(training_labels) > 0
+                existing.training_labels = training_labels
+            existing.use_for_training = len(training_labels) > 0
         session.flush()
 
 
+def load_csv(path: str) -> Iterator[List[str]]:
```
Comment (bdittes) [inline local.py]: This appears unused

File: app/models.py [hunk 3]
```diff
@@ -78,6 +78,7 @@ class __mongometa__:
     _id = FieldProperty(schema.ObjectId)
     model = FieldProperty(schema.String)
     seq = FieldProperty(schema.String)
+    sharedId = FieldProperty(schema.String)
```
Comment (bdittes) [inline app/models.py]: If you want to do something with this in the future, it's probably a good idea to add an index for it (indexes = ... above)

File: app/classifier.py (hunk 1 above)
Comment (Tilana) [reply app/classifier.py]: I changed the batch size from being a constant to an optional argument.
This way it's easier to adapt it from a higher level.

File: local.py (hunk 2 above)
Comment (Tilana) [reply local.py]: good catch!

File: app/models.py (hunk 3 above)
Comment (Tilana) [reply app/models.py]: I think an index is not really needed as it's more an additional property which enables to relate other metadata in Uwazi or a seperate csv file to a classification sample.
Also, sharedIds might not be unique. They could refer to a translations of a text or simply occur by chance when having classification samples from different collections/models, etc.

**Pull request merged**